presto-hive/src/main/java/io/prestosql/plugin/hive/util/HiveTypeTranslator.java

```diff
@@ -64,7 +64,7 @@
 {
     private HiveTypeTranslator() {}
 
-    public static TypeInfo translate(Type type)
+    public static TypeInfo toTypeInfo(Type type)
```

Comment:
the name of the class is "translator", so "translate" is not a bad name, is it?

Reply:
I renamed that because I added other translation methods to the class.